back/sssscs/src/main/java/com/ib/user/UserController.java

```diff
@@ -0,0 +1,53 @@
+package com.ib.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ib.user.dto.UserCreateDto;
+import com.ib.user.dto.UserLoginDto;
+
+import jakarta.annotation.security.PermitAll;
+
+@RestController
+public class UserController {
+	@Autowired
+	private UserService userService;
+	@Autowired
+	private AuthenticationManager authManager;
+	
+	@PostMapping("/api/user")
+	public ResponseEntity<?> register(@RequestBody UserCreateDto dto) {
+		User user = userService.register(dto);
```

Comment:
Don't use ? when you know return type

Reply:
Changed in [f8f5546](https://github.com/magley/sssscs/pull/1/commits/f8f55465c6d87638c36cb8e8d2d6f203bb7598f4).